client/src/components/Reviews_Ratings/Ratings_Reviews.jsx

```diff
@@ -1,13 +1,36 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import axios from 'axios';
+import API_KEY from '../../../../config.js';
```

Comment:
if your config.js is in your root directory, it can be accessed with the following file path: '/config.js'